lib/dash/dash_parser.js

```diff
@@ -1073,8 +1073,10 @@ shaka.dash.DashParser = class {
       return element.getAttribute('schemeIdUri') == expectedUri &&
           element.getAttribute('value') == expectedValue;
     });
+    let spatialAudio = false;
     if (hasJoc) {
       context.representation.mimeType = 'audio/eac3-joc';
```

Comment:
Some things to confirm - true or false:
- the JOC attribute can only be present on audio streams
- if it is present, that means it's spatial audio
- spatial audio always has 'audio/eac3-joc' mime type

Reply:
the JOC attribute can only be present on audio streams --> True
if it is present, that means it's spatial audio --> True
spatial audio always has 'audio/eac3-joc' mime type --> False, There are other formats with spatial audio (eg: dts:x)

Reply:
Got it, so for this PR you're only dealing with the JOC ones, then?

Reply:
Right now yes, but in the future other formats can be added.